dev-src/dev/db.clj

```diff
@@ -18,8 +18,11 @@
   (db/transact seed-tx-data))
 
 (defn reset-db! []
-  (d/delete-database (config :datomic :uri))
-  (db/initialize)
+  ;; TODO: This isn't really correct. In Datomic you shouldn't delete and
```

Comment:
This makes me cringe: the comment tells me it's wrong, but not how it's wrong, so I'm not sure how worried I should be.